### PR TITLE
[MRG+2] Added threshold option for find_eog_events() and create_eog_epochs()

### PR DESCRIPTION
**Pull request by pmolfese**
#### What does this implement/fix?
Found that find_eog_events() and thus create_eog_epochs() weren't detecting many EOG events, despite seeing them in data.  The function used on the backend - peak_finder() - can take a threshold as argument, added option for user to pass a threshold to the function.  Initial testing found better detection of blinks.  

#### Additional information
Perhaps more necessary in EEG recordings.

### CODE REVIEW

**Comment by agramfort** [comment]
thanks @pmolfese 

is there any chance to add a tiny test?

You'll need to update what's new API section to document the new parameter.

**Comment by pmolfese** [comment]
Added a very basic test to test_eog.py.  

Apologies for the basic question: Where is the API document?

**Comment by agramfort** [comment]
here :

https://github.com/mne-tools/mne-python/blob/master/doc/whats_new.rst#api

File: mne/preprocessing/tests/test_eog.py [hunk 1]
```diff
@@ -20,3 +20,7 @@ def test_find_eog():
 
     events = find_eog_events(raw, reject_by_annotation=True)
     assert all(events[:, 0] < 29000)
+
+	#threshold option 
+	events_thr = find_eog_events(raw, thresh=100e-6)
+	assert len(events) == 5
```
Comment (agramfort) [inline mne/preprocessing/tests/test_eog.py]: looks like you have an indentation problem:

did you run:

`pytest mne/preprocessing/tests/test_eog.py`

?

and

`flake8 mne/preprocessing/tests/test_eog.py`

?

Comment (pmolfese) [reply mne/preprocessing/tests/test_eog.py]: Sorry about that, fixed now!

**Comment by agramfort** [comment]
@pmolfese you still have flake8 errors

```
$ flake8 mne/preprocessing/eog.py
mne/preprocessing/eog.py:42:5: W191 indentation contains tabs
mne/preprocessing/eog.py:42:5: E101 indentation contains mixed spaces and tabs
mne/preprocessing/eog.py:109:80: E501 line too long (81 > 79 characters)
mne/preprocessing/eog.py:111:80: E501 line too long (82 > 79 characters)
mne/preprocessing/eog.py:161:80: E501 line too long (90 > 79 characters)
mne/preprocessing/eog.py:218:1: W293 blank line contains whitespace
mne/preprocessing/eog.py:220:5: W191 indentation contains tabs
mne/preprocessing/eog.py:220:5: E101 indentation contains mixed spaces and tabs
mne/preprocessing/eog.py:243:80: E501 line too long (86 > 79 characters)
```

**Comment by pmolfese** [comment]
Fixed flake errors in eog.py.  Still getting used to Python style.

**Comment by agramfort** [comment]
thanks @pmolfese

**Pull request merged**